types/common_label.go

```diff
@@ -24,6 +24,7 @@ const (
 	LabelTraefikFrontendWhitelistSourceRange     = LabelPrefix + "frontend.whitelistSourceRange"
 	LabelBackend                                 = LabelPrefix + "backend"
 	LabelBackendID                               = LabelPrefix + "backend.id"
+	LabelBackendPort                             = LabelPrefix + "backend.port"
```

Comment:
Could you please use [`types.LabelPort`](https://github.com/containous/traefik/blob/master/types/common_label.go#L10) instead of creating a new label.